app/View/Composers/TopNavComposer.php

```diff
@@ -74,11 +74,14 @@ public function compose(View $view): void
                 fn () => $user->peers()->where('active', '=', 1)->where('seeder', '=', false)->count(),
             ),
             'hasActiveWarning'    => $user->warnings()->where('active', '=', true)->exists(),
-            'hasUnresolvedReport' => $user->group->is_modo && Report::query()->whereNull('snoozed_until')->where(
-                'solved',
-                '=',
-                false
-            )->exists(),
+            'hasUnresolvedReport' => $user->group->is_modo && Report::query()
```

Comment:
the index `reports_solved_snoozed_until_index` currently indexes the columns (`solved`, `snoozed_until`), but with the new query for the top nav, you'll want to remove that index and add another one on (`solved`, `snoozed_until`, `staff_id`)